coalib/processes/Processing.py

```diff
@@ -459,13 +460,15 @@ def yield_ignore_ranges(file_dict):
                                    1,
                                    line_number,
                                    len(file[line_number-1])))
-                elif 'ignore ' in line:
-                    end_line = min(line_number + 1, len(file))
-                    yield (get_ignore_scope(line, 'ignore '),
-                           SourceRange.from_values(
-                               filename,
-                               line_number, 1,
-                               end_line, len(file[end_line - 1])))
+
+                for ignore_stmt in ['ignore ', 'noqa ', 'noqa']:
```

Comment:
I think this should happen in the `else` case to achieve same codeflow like before :)
(also yields slightly performance)